gitbook-docs/market/cmd-reference/user.md

```diff
@@ -26,33 +24,56 @@ options:
                         password
   -e EMAIL, --email EMAIL
                         email
-
 ```
 
-Ex:
+## Example
 
-```
+```bash
 python3 signup.py -u helloworld -p password123 -e hello@test.com
 ```
 
-Output:
+## Possible results
 
-### **Success**
+Success:
 
-```
+```json
 {"user":"helloworld","active":true,"extra":{},"error":false,"code":201}
 ```
 
-### **Error#1: Invalid username format**
+Error #1 — invalid username format:
 
-```
+```bash
 Error: 400 {"code":409,"error":true,"errorMessage":"Inavlid format for username"}
 ```
 
-The username does not support special characters and can only be created with a combination of alphabets and numbers.
+The username does not support special characters and can only be a combination of letters
+and numbers.
 
-**Error#2: Duplicate user**
+Error #2 — duplicate user:
 
-```
+```bash
 Error: 500 {"error":true,"errorNum":1702,"errorMessage":"duplicate user","code":500}
 ```
+## register as a proof producer
+
+If you wish to submit generated proofs to the Proof Market,
+you need to also register as a proof producer:
+
+```console
+foo@bar:~$ python3 signup.py producer --url https://example.com
+```
+
+his will return producer metadata in JSON format:
+
+```json
+{
+  "_key": "...",
+  "_id": "producer/...",
+  "_rev": "_fneT7J2---",
+  "description": "Generic Producer",
+  "url": "https://example.com",
+  "name": "username",
+  "createdOn": 1677588821180,
+  "updatedOn": 1677588821180
+}
+```
```

Comment:
This is a separate issue. https://github.com/NilFoundation/proof-market-toolchain/issues/83

Reply:
Removed this part, saved it in #83